src/lightning/pytorch/loggers/neptune.py

```diff
@@ -43,6 +43,26 @@
 _INTEGRATION_VERSION_KEY = "source_code/integrations/pytorch-lightning"
 
 
+def _sanitize_expected_model_path(dir_path: str) -> str:
+    """Sanitize the expected model path so that it is correct irrespective of the underlying os.
+
+    >>> _sanitize_expected_model_path('my_model/checkpoints//')  # unix-like path separator os
+    'my_model/checkpoints/'
+    >>> _sanitize_expected_model_path('my_model/checkpoints')
+    'my_model/checkpoints/'
+
+    >>> _sanitize_expected_model_path('my_model\\checkpoints')  # windows
+    'my_model\\checkpoints\\'
+
+    """
```

Comment:
```suggestion
    """Sanitize the expected model path so that it is correct irrespective of the underlying OS.

    Args:
        dir_path:
            Path of the model.

    Returns:
        Sanitized version of the path.

    Examples:
        >>> _sanitize_expected_model_path('my_model/checkpoints//')  # OS with Unix-like path separator
        'my_model/checkpoints/'
        >>> _sanitize_expected_model_path('my_model/checkpoints')
        'my_model/checkpoints/'
        >>> _sanitize_expected_model_path('my_model\\checkpoints')  # Windows
        'my_model\\checkpoints\\'

    """
```